### ISSUE TITLE
Support scrollbars for LazyColumn with reverseLayout = true

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#209]
```
Box {
    val state = rememberLazyListState()
    val itemCount = 100
    val itemHeight = 20.dp

    LazyColumn(state = state, reverseLayout = true) {
        items((1..itemCount).toList()) { x ->
            Text(x.toString(), Modifier.graphicsLayer(alpha = 0.5f).height(itemHeight))
        }
    }

    VerticalScrollbar(
        rememberScrollbarAdapter(state, itemCount, itemHeight),
        Modifier.align(Alignment.CenterEnd)
    )
}
```

For now (0.3.0-build136) scrollbar in this case doesn't work as expected.

**Comment by igordmn** [comment]
Fixed in 0.4.0-build210.

Added a parameter `reverseLayout` to `VerticalScrollbar`/`HorizontalScrollbar`

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6422">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
